src/user-federation/ldap/mappers/LdapMapperRoleGroup.tsx

```diff
@@ -52,6 +56,32 @@ export const LdapMapperRoleGroup = ({
     []
   );
 
+  useFetch(
+    async () => {
+      if (id) {
+        return await adminClient.components.findOne({ id });
+      }
+      return undefined;
+    },
```

Comment:
According to the route type the `id` field is always defined, which means this code can be simplified.
```suggestion
    () => adminClient.components.findOne({ id }),
```